### ISSUE TITLE
Things to add to documentation.

### ISSUE THREAD

**Issue by pcklink** [afids/afids-macaca#3]
- Manually set slice spacing in Slicer to something low for fine control of fiducial placement
- 
-

**Comment by jclauneuro** [comment]
Thanks for your feedback. I've never used the set slice spacing feature. Would be great if you could provide a demo on how to use that effectively. This can certainly be added to the doc.

Once we've finalized the protocol, we will update with better images. Probably using NMTv1.3 as the example as you've indicated.

**Comment by pcklink** [comment]
<img width="711" alt="Screenshot 2020-04-09 20 47 35" src="https://user-images.githubusercontent.com/6423796/78929856-64f11300-7aa3-11ea-9ef5-ddd42d5585e5.png">

This is where you can set how finely/coarsely Slicer moves through slices.

**Comment by greydongilmore** [comment]
I have been working and developing in 3D Slicer and I never knew this was an option that a user could set without python code! Will definitely add this to the docs.

**Comment by bmahmou** [comment]
Should we also include fiducial scale adjustment? the default is ok but it can make a difference in accuracy when placing the points

**Comment by pcklink** [comment]
Probably a good idea to try to keep that uniform. I made them smaller myself.

**Comment by pcklink** [comment]
The protocol could have a clearer starting point
-- ready to get started takes you to AFIDs protocol >> skipping the linear transform
	-- have a steps list (numbered?)

**Comment by jclauneuro** [comment]
Agree, the AC-PC transform should be the first step of the AFIDs protocol probably with a small blurb about the role of that transformation (why is it important) and perhaps some historical background. This issue is not specific to afids-macaca and can probably be moved to afids-protocol.
